Готова вторая проектная работа

vladimirton opened the pull request.

vladimirton committed:
update

vladimirton committed:
update

vladimirton committed:
update

Mililiya commented on index.html:
```diff
@@ -1,10 +1,185 @@
+		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<title>Оно тебе надо — аукцион вещей, в которые никто не верил</title>
+		<link rel="stylesheet" href="fonts/fonts.css" />
```
**Надо исправить:**

Относительный путь необходимо начинать с ./, это указание на текущий каталог, откуда идет обращение к подключаемому файлу, без этого указания могут возникнуть проблемы с подключением в некоторых системах и браузерах

**Проконтролируйте это тут и далее на странице**

vladimirton replied:
Исправлено в коммите bfeb1ad

Mililiya commented on index.html:
```diff
@@ -1,10 +1,185 @@
+	</head>
+	<body>
+		<header class="header">
+			<nav class="header__menu">
```
**Отлично:**

Отлично, что использовали тег `nav` для списка ссылок навигации

Mililiya commented on index.html:
```diff
@@ -1,10 +1,185 @@
+	<body>
+		<header class="header">
+			<nav class="header__menu">
+				<ul class="header__links-list">
```
**Отлично:**

Отличное использование тега списка `ul > li`

Mililiya commented on index.html:
```diff
@@ -1,10 +1,185 @@
+				<img class="header__logo-image" src="images/logo-black.svg" alt="Логотип" />
+			</a>
+
+			<address class="address header__address">
```
**Отлично:**

Отлично, что использовали тег `address`

Mililiya commented on index.html:
```diff
@@ -1,10 +1,185 @@
+			</address>
+		</header>
+
+		<main>
```
**Отлично:**

Отлично, что для основного содержимого страницы использовали тег `main`

Mililiya commented on index.html:
```diff
@@ -1,10 +1,185 @@
+		<main>
+			<section class="cover">
+				<div class="overlay"></div>
+				<h1 class="cover__title">
```
**Отлично:**

Верно подобрали уровень заголовка, отлично что `h1` только один в проекте

Mililiya commented on index.html:
```diff
@@ -1,10 +1,185 @@
+				</h1>
+
+				<div class="cover__description">
+					<h2 class="cover__description-text">Аукцион вещей, в которые никто не верил</h2>
```
**Можно лучше:**

Для подзаголовков лучше применять тег `<p>`, т.к. через теги `<h1>—<h6>` сайт настраивается для выдачи в поисковиках по ключевым запросам пользователя, их лучше применять когда необходимо разделить большой объем текста, например, как это делают в книгах

vladimirton replied:
Исправлено в коммите 1dc05f8

Mililiya commented on index.html:
```diff
@@ -1,10 +1,185 @@
+			</section>
+
+			<section class="lots">
+				<h3 class="lots__heading">Лоты</h3>
```
**Надо исправить:**

Тут нужно использовать заголовок уровня `h2`, так как это главный заголовок секции

[Подробнее тут](https://doka.guide/html/h1-h6/)

vladimirton replied:
Исправлено в коммите 1dc05f8

Mililiya commented on index.html:
```diff
@@ -1,10 +1,185 @@
+
+			<section class="lots">
+				<h3 class="lots__heading">Лоты</h3>
+				<ul class="lots__card-list">
```
**Отлично:**


Для однотипных элементов используется тег списка

Mililiya commented on index.html:
```diff
@@ -1,10 +1,185 @@
+				<ul class="lots__card-list">
+					<li class="lots__card-list-item">
+						<a class="card-link">
+							<article class="card card_type_film">
```
**Отлично:**


Отлично, `article` тут как раз подходит

Mililiya commented on index.html:
```diff
@@ -1,10 +1,185 @@
+						<a class="card-link">
+							<article class="card card_type_film">
+								<div class="overlay"></div>
+								<h4 class="card__title">Фильм режиссёра, который бросил киношколу</h4>
```
**Надо исправить:**

Следует соблюдать уровни заголовков, согласно структуре страницы тут следует использовать заголовок уровня `h3`, [подробнее](https://doka.guide/html/h1-h6/)

Проконтролируйте это тут и далее в блоке

vladimirton replied:
Исправлено в коммите 1dc05f8

Mililiya commented on index.html:
```diff
@@ -1,10 +1,185 @@
+		</main>
+
+		<footer class="footer">
+			<address class="address">
```
**Отлично:**

Отлично, что использовали тег `address`

Mililiya commented on index.html:
```diff
@@ -1,10 +1,185 @@
+						<img class="about__logo-image" src="../images/logo-white.svg" alt="Логотип" />
+					</div>
+					<div class="about__content">
+						<h4 class="about__title">Об аукционе</h4>
```
**Надо исправить:**

Тут нужно использовать заголовок уровня `h2`, так как это главный заголовок секции

[Подробнее тут](https://doka.guide/html/h1-h6/)

vladimirton replied:
Исправлено в коммите 1dc05f8

Mililiya commented on index.html:
```diff
@@ -1,10 +1,185 @@
+
+		<footer class="footer">
+			<address class="address">
+				<a class="header__link" href="tel:+99995555555">+9 999 555 5555</a><br />
```
**Надо исправить:** 

Контактная информация полностью одинаковая в блоках `header` и `footer`, классы так же должны переиспользоваться (или в данном случае их не должно быть, так же как и в `header`)

vladimirton replied:
Исправлено 7891a89

Mililiya commented on index.html:
```diff
@@ -1,10 +1,185 @@
+				<span>наб. Принсенграхт 263- 265, Амстердам</span>
+			</address>
+
+			<nav class="footer__menu">
```
**Отлично:**

Отлично, что использовали тег `nav` для списка ссылок навигации

Mililiya commented on index.html:
```diff
@@ -1,10 +1,185 @@
+			</address>
+
+			<nav class="footer__menu">
+				<ul class="footer__menu-list">
```
**Отлично:**

Отличное использование тега списка `ul > li`

Mililiya commented on index.html:
```diff
@@ -1,10 +1,185 @@
+				</ul>
+			</nav>
+
+			<ul class="footer__social-list">
```
**Отлично:**

Отличное использование тега списка `ul > li`

Mililiya commented on index.html:
```diff
@@ -1,10 +1,185 @@
+				</li>
+				<li class="footer__social-list-item">
+					<a class="footer__social-link" href="#0">
+						<img class="footer__social-icon" src="images/pinterest.svg" alt="pinterest" />
```
**Отлично:**

Отлично, что не забываете указывать `alt` у изображений

Mililiya commented on styles/style.css:
```diff
@@ -1 +1,337 @@
+.address {
+	font-style: normal;
+	font-size: 18px;
+	width: 240px;
```
**Можно лучше:** 

Для задания ширины чаще всего лучше использовать свойство `max-width`, а не `width`. В отличии от `width`,  `max-width` позволит блоку уменьшить свою ширину, например если страница будет отображаться в окне с маленькой шириной. Это делает верстку более адаптивной

vladimirton replied:
Исправлено: 95c3743

Mililiya commented on styles/style.css:
```diff
@@ -1 +1,337 @@
+
+.overlay {
+	position: absolute;
+	background-color: rgb(0 0 0 / 0.4);
```
**Надо исправить:**

Прозрачность для линзы следует установить по макету

vladimirton replied:
Исправлено: 95c3743

Mililiya commented on styles/style.css:
```diff
@@ -1 +1,337 @@
+.aligned-text_center {
+	text-align: center;
+	position: relative;
+	left: -30px;
```
**Надо исправить:**

Координаты принято использовать в связке с `position: absolute;`, а абсолютное позиционирование должно использоваться только там, где нельзя применить статичное или относительное позиционирование. В данном случае координаты следует удалить, их можно заменить на `margin/padding`

vladimirton replied:
Исправлено: 95c3743

Mililiya commented on styles/style.css:
```diff
@@ -1 +1,337 @@
+	border: 1px solid #fff;
+	color: #fff;
+	text-align: center;
+	font-family: Raleway;
```
**Надо исправить:**

Использовать ```font-family: 'Шрифт'```,  является не совсем корректным решением. 
 В том случае, если по какой-либо причине шрифты не загрузятся или будет использоваться браузер, не поддерживающий текущий формат шрифта - следует предусмотреть такой исход и добавить системные шрифты. 
 Например, sans-serif

Проконтролируйте это тут и далее в файле

vladimirton replied:
Исправлено: 95c3743

Mililiya commented on styles/style.css:
```diff
@@ -1 +1,337 @@
+	align-items: center;
+	padding: 51px 40px 48px 40px;
+}
+.card_type_film {
```
**Надо исправить:**

У карточек есть повторяющиеся правила, их следует установить общему блоку `card` чтобы избежать повторения у селекторов. У модификаторов должно остаться только одно правило - разное фоновое изображение

vladimirton replied:
Исправлено: 47bfd61

Mililiya commented on styles/style.css:
```diff
@@ -1 +1,337 @@
+	letter-spacing: 0.2px;
+}
+
+.footer {
```
**Надо исправить:**

https://disk.yandex.ru/i/hNgdFKSXptOj2A текстовые элементы следует стилизовать по макету

vladimirton replied:
Исправлено: 233d1bb

Mililiya commented on styles/style.css:
```diff
@@ -1 +1,337 @@
+	padding-top: 101px;
+}
+
+.about__text {
```
**Надо исправить:**

https://disk.yandex.ru/i/Xhd3_2UXvUw_6w при добавлении абзаца расстояние между текстом не должно схлопываться

vladimirton replied:
Исправлено: 233d1bb

Mililiya reviewed (commented):
Здравствуйте, Владимир. 
Нужно развернуть общий комментарий, что бы увидеть его полностью ↓

Была проделана большая работа и она выполнена на хорошем уровне!

Что понравилось в работе:

- Отлично структурированный `html` код, страница валидна
- Страница соответствует макету
- Аккуратное форматирование кода
- Стили и изображения в собственных файлах и папках
- Используются технологии flex и grid.

Но есть так же и замечания, которые потребуют доработки.
Места, которые надо исправить в проекте:

- Комментарии в коде, отмеченные как "Надо исправить"

Что можно улучшить:

- Комментарии в коде, отмеченные как "Можно лучше"

Обратите внимание, комментарии "Надо исправить" обязательны для исправления, работа будет зачтена после исправления всех таких комментариев. Комментарии "Можно лучше" носят рекомендательный характер, их выполнение остается на Ваше усмотрение.

Вносите правки без спешки, следите за тем, что изменилось после внесения той или иной правки и при этом больше ничего не сломалось. Убедитесь перед отправкой, что все правки внесены, основные требования соответствуют брифу и чек листу. Если появятся трудности - обращайтесь к наставнику и старшему студенту, они Вам помогут.

Успешной работы над проектом, и до встречи на следующем ревью!

vladimirton committed:
Исправлены относительные пути - см. обсуждение на https://github.com/vladimirtton/ono-tebe-nado/pull/1#discussion_r1721032014

vladimirton committed:
Исправлена иерархия заголовков: https://github.com/vladimirtton/ono-tebe-nado/pull/1\#discussion_r1721032448, https://github.com/vladimirtton/ono-tebe-nado/pull/1\#discussion_r1721032472, https://github.com/vladimirtton/ono-tebe-nado/pull/1\#discussion_r1721032649, https://github.com/vladimirtton/ono-tebe-nado/pull/1\#discussion_r1721032864

vladimirton committed:
Убрал лишний класс https://github.com/vladimirtton/ono-tebe-nado/pull/1\#discussion_r1721033337

vladimirton committed:
Исправления и улучшения:
- Исправлена ширина для адаптивности: https://github.com/vladimirtton/ono-tebe-nado/pull/1#discussion_r1721034002
- Уточнена прозрачность элементов: https://github.com/vladimirtton/ono-tebe-nado/pull/1#discussion_r1721034088
- Замена координат на padding для лучшей адаптивности: https://github.com/vladimirtton/ono-tebe-nado/pull/1#discussion_r1721034811
- Добавлен альтернативный шрифт для улучшения доступности: https://github.com/vladimirtton/ono-tebe-nado/pull/1#discussion_r1721034946

vladimirton committed:
Исправления и улучшения:
- Убрано схлопывание между абзацами: https://github.com/vladimirtton/ono-tebe-nado/pull/1#discussion_r1721035426
- Испрвлена стилизация футера: https://github.com/vladimirtton/ono-tebe-nado/pull/1#discussion_r1721035284

vladimirton committed:
Исправления и улучшения:
- Испрвлена стилизация карточек: https://github.com/vladimirtton/ono-tebe-nado/pull/1#discussion_r1721035070
- Незначительные правки

vladimirton committed:
Незначительные правки

mikebars1995 reviewed (commented):
Поздравляю! Ваша работа принята.

Вы отлично потрудились.

Удачного дальнейшего обучения.
